modules/services/tldr-update.nix

```diff
@@ -0,0 +1,50 @@
+{ config, lib, pkgs, ... }:
+let cfg = config.programs.tldr-update;
+in {
+  meta.maintainers = [ lib.maintainers.perchun ];
+
+  options.programs.tldr-update = {
```

Comment:
Looks like this was moved to services but is still optioned under programs.

Reply:
Oh, you're right. Fixed that